test/xsd/base64.xsd

```diff
@@ -1,4 +1,7 @@
-<schema xmlns="http://www.w3.org/2001/XMLSchema" xmlns:here="http://example.org/" targetNamespace="http://example.org/">
+<schema xmlns="http://www.w3.org/2001/XMLSchema" xmlns:here="http://example.org/"
```

Comment:
Please update expected generation code of this xsd for other supported languages to fix unit tests build break.